src/components/InLineNotification.tsx

```diff
@@ -9,12 +9,13 @@ export enum Severity {
   Informational,
   Warning,
   Error,
+  SoftInformational,
 }
 
 interface Props {
   severity: Severity
   title?: string | null
-  description: string
+  description?: string | JSX.Element | null
```

Comment:
Any reason to make this optional? Seems like the current usage you're adding should always pass a JSX element. Also is there a way to limit the type of JSX.Element? The typing seems too broad. What if a <View> for example is passed to this?